pkg/build/definition.go

```diff
@@ -17,26 +17,27 @@ type Definition struct {
 // imageData contains any scripts, metadata, etc... that needs to be
 // present in some from in the final built image
 type imageData struct {
-	metadata []byte
-	labels   []string
+	Metadata []byte
```

Comment:
What form is the metadata going to take? Is it going to be completely free-form, or JSON?. If later can we use a json.RawMessage just to signify intent?

Reply:
Yea I'm pretty sure it'll just be raw JSON, not sure how to use `json.RawMessage` but I'll take a look